<ui version="4.0" >
 <class>TrainMain</class>
 <widget class="QWidget" name="TrainMain" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>711</width>
    <height>443</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Training</string>
  </property>
  <property name="windowIcon" >
   <iconset>
    <normaloff/>
   </iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <widget class="QTableWidget" name="twTrainingWords" >
     <property name="selectionMode" >
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior" >
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <column>
      <property name="text" >
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Seiten</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Relevanz</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="KPushButton" name="pbTrainText" >
     <property name="font" >
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="toolTip" >
      <string>Startet das Training des selektierten Textes.</string>
     </property>
     <property name="layoutDirection" >
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text" >
      <string>   (Markierten Text &amp;vorlesen (trainieren</string>
     </property>
     <property name="iconSize" >
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="KPushButton" name="pbDelText" >
       <property name="toolTip" >
        <string>Löschen des markierten Textes aus dem System</string>
       </property>
       <property name="text" >
        <string>  Markierten Text &amp;löschen</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KPushButton" name="pbImportText" >
       <property name="toolTip" >
        <string>Importiert neue Texte aus dem Internet oder aus Dateien</string>
       </property>
       <property name="text" >
        <string>  Text &amp;importieren</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KPushButton" name="pbImportDir" >
       <property name="toolTip" >
        <string>Importiert neue Texte aus dem Internet oder aus Dateien</string>
       </property>
       <property name="text" >
        <string>Trainingsdaten Importieren</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
